### PR TITLE
Fix date formatting in Latvian

### PR DESCRIPTION
**Pull request by tan75**
As per
https://unicode.org/cldr/cldr-aux/charts/32/summary/lv.html#1882

### CODE REVIEW

File: src/locale/lv/_lib/formatLong/index.js [hunk 1]
```diff
@@ -1,41 +1,41 @@
 import buildFormatLongFn from '../../../_lib/buildFormatLongFn/index'
 
 var dateFormats = {
-  full: "y. 'gada' M. MMMM., EEEE",
-  long: "y. 'gada' M. MMMM",
+  full: "y. 'gada' d. MMMM, EEEE",
```
Comment (gkuznets) [inline src/locale/lv/_lib/formatLong/index.js:4]: Actually, according to this https://unicode.org/cldr/cldr-aux/charts/32/summary/lv.html#1882
format should be "EEEE, y. 'gada' d. MMMM".

Comment (tan75) [reply src/locale/lv/_lib/formatLong/index.js:4]: > Actually, according to this https://unicode.org/cldr/cldr-aux/charts/32/summary/lv.html#1882
> format should be "EEEE, y. 'gada' d. MMMM".

Yes, you are right. Will fix it shortly! Thanks!

Comment (tan75) [reply src/locale/lv/_lib/formatLong/index.js]: @gkuznets
should be good now

Comment (gkuznets) [reply src/locale/lv/_lib/formatLong/index.js]: Thanks!

**Review by gkuznets — APPROVED** [review]

**Review by kossnocorp — APPROVED** [review]
🙏

**Pull request merged**